lib/challenge.js

```diff
@@ -51,7 +85,15 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+var some_pet = {
+  name: 'Natty',
+  species: 'yellow lab',
+  breed: 'pure',
+  noise: 'bark',
+  makeNoise: function (){
+    return this.noise;
+  }
+}
```

Comment:
Awesome. Great method, it is general and relies on the object to specify the noise string. Full credit.